python/pylibraft/pylibraft/solver/lanczos.pyx

```diff
@@ -0,0 +1,265 @@
+#
+# Copyright (c) 2024-2024, NVIDIA CORPORATION.
```

Comment:
Please do the following search and replace across the codebase: `%s/2024-2024/2024`